src/playout/mod.rs

```diff
@@ -74,21 +75,23 @@ pub trait Playout: Sync + Send {
         size as usize * size as usize * 3
     }
 
-    fn select_move(&self, board: &Board) -> Move {
+    fn select_move(&self, board: &Board, rng: &mut XorShiftRng) -> Move {
         let color = board.next_player();
         let vacant = board.vacant();
-        let playable_move_exists =  vacant
+        let playable_move =  vacant
```

Comment:
What was the reason behind this change? I didn't see any speed increase and it seems to have made the code more complex. To be fair, the thing with the pass move I added here didn't help either. ;)